tools/MultiProjectValidator/AnalysisRules/DependencyMismatch/DependencyInfo.cs

```diff
@@ -9,7 +10,7 @@ public static DependencyInfo Create(ProjectContext context, LibraryDescription l
             return new DependencyInfo
             {
                 ProjectPath = context.ProjectFile.ProjectFilePath,
-                Version = library.Identity.Version.ToString(),
+                Version = library.Identity.Version?.ToString() ?? String.Empty,
```

Comment:
`mscorlib` was one I think. Framework assemblies don't have to have version numbers attached to them because they get their version from the associated TFM.